src/components/DateTimeDisplay.tsx

```diff
@@ -0,0 +1,18 @@
+import { formatDateTime } from 'utils/dateFormatter';
+
+interface DateTimeDisplayProps {
+  date: string;
+};
+
+const DateTimeDisplay = ({ date } : DateTimeDisplayProps) => {
+  const { formattedDate, formattedTime } = formatDateTime(date);
```

Comment:
is there a specific format in which we are converting date and name function as that